document-generator-core/src/main/java/uk/gov/companieshouse/document/generator/core/document/render/RenderedDocumentJsonHandler.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.companieshouse.document.generator.core.document.render;
+
+import uk.gov.companieshouse.document.generator.core.document.models.DocumentGenerationCompleted;
+
+public interface RenderedDocumentJsonHandler {
```

Comment:
yes, as there is a possibility this may change in the future, and it's good practice to program to interfaces in general. Granted some may argue that there are occasions where it is not needed. However, as I am not sure if this will change, it's better to decouple from the implementations to make it easier to change later.

Reply:
reason I ask it the class name sounds like a very specific implementation.... be careful about adding code clutter if you arn't going to need it!